bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java

```diff
@@ -2233,7 +2233,7 @@ public static boolean joinRandomArena(Player p) {
 
     public static List<IArena> getSorted(List<IArena> arenas) {
         List<IArena> sorted = new ArrayList<>(arenas);
-        sorted.sort(new Comparator<IArena>() {
```

Comment:
was that redundant? what's the reason here?

Reply:
In java 11 you don't need to put what type of array it is as at compile time it knows the type of array

Reply:
So just makes the code look a bit cleaner but I can always add it back